doc/architecture.md

```diff
@@ -0,0 +1,136 @@
+Architecture
+============
+
+This section describes how a Management Pack works, and how it interacts with the VMware 
+Aria Operations Collector.
+
+## Management Pack
+A management pack consists of content, metadata, and an adapter.
+
+* Content includes Dashboards, Reports, Icons, Symptoms, Alerts, Traversals, others.
```

Comment:
Each type of content could be a hyperlink to an explanation of what it is. 

Same with describe.xml and manifest.xml.

Reply:
We don't really have pages for the describe.xml or manifest.txt files to link to, but I've added links to the relevant content sections.